feat: FAQ - Add Migration Chapter (#573)
lukso-network/docs: docs/faq/migration/asset-migration.md

---
sidebar_label: 'Asset Migration'
sidebar_position: 2
---

# Asset Migration

> ### Will previous Universal Profiles be migrated to mainnet?

Reviewer: ```suggestion
### Will Universal Profiles previously created on L14 be migrated to mainnet?
```

> LUKSO will migrate all Universal Profiles previously created on the L14 Testnet by [The Dematerialised Marketplace](https://thedematerialised.com/) or [Universal Profile Explorer](https://universalprofile.cloud/). If you deployed custom profiles on older testnets, you would be responsible for migrating them to the mainnet.

Reviewer 2: there are more we will migrate, i dont have the list here but there are more.

Author: There are more, but they were all deployed on L14 profile explorer no? Is anyone to ask for this? I just took the question from the old FAQ board. @Hugoo

Reviewer 2: Ok then let's keep it like this for now ;)

> LUKSO will migrate all Universal Profiles previously created on the L14 Testnet by [The Dematerialised Marketplace](https://thedematerialised.com/) or [Universal Profile Explorer](https://universalprofile.cloud/). If you deployed custom profiles on older testnets, you would be responsible for migrating them to the mainnet.

Reviewer: ```suggestion
LUKSO will migrate all Universal Profiles previously created on the L14 Testnet by [The Dematerialised Marketplace](https://thedematerialised.com/) or [Universal Profile Explorer](https://universalprofile.cloud/). If you deployed custom profiles on older testnets, you would be responsible for migrating them yourself to the mainnet.
```

> ### Will previously aquired NFTs be migrated to mainnet?

Reviewer: ```suggestion
### Will previously acquired NFTs be migrated to mainnet?
```

> LUKSO will migrate all previously issued NFTs by the [LUKSO](https://lukso.network/) and [The Dematerialised](https://thedematerialised.com/) brands. If you deployed custom assets and services on older testnets, you would be responsible for migrating them to the mainnet.

Reviewer 2: there are more link Out of the Ether NFT, and other ones.

Author: Yes, but OOE was done in collaboration with LUKSO. I will update both, to say we will migrate those with official collaborations

Reviewer 2: ok
